### ISSUE TITLE
DUIK RC 2 Mac Zip file not able to unzip

### ISSUE THREAD

**Issue by reneandritsch** [Rainbox-dev/DuAEF_Duik#239]
### Tool & Version
- Duik 16.0.0-RC2

### OS

- Mac OS 10.13.5

# What's the issue?
I’m not bale to unzip the Mac archive

### Error Message
![unzip_error](https://user-images.githubusercontent.com/38455050/42133888-a2defe50-7d31-11e8-94f3-2de36d09dd63.png)

**Comment by reneandritsch** [comment]
Nicholas, the link worked. Thank you. The zipped file from github was also slighty bigger 20,2 MB compared to 20 MB.

**Comment by reneandritsch** [comment]
The ffmpeg is still only for WINDOWS, correct?

**Comment by reneandritsch** [comment]
The startup is now blazingly fast, Amazing!

**Comment by Nico-Duduf** [comment]
No, ffmpeg now works on Mac OS too, I've been able to finish and test 
this these days, and it seems to work well.
For now it's only used for exports to Audition, but it's just a first 
step... :)

Le 01/07/2018 à 13:35, reneandritsch a écrit :

**Comment by reneandritsch** [comment]
Incredible how much work you get done! Thanks!
